templates/confirmation/confirm_email_change.html

```diff
@@ -0,0 +1,32 @@
+{% extends "zerver/portico.html" %}
+
+{% block portico_content %}
+
+<div class="pitch">
+    <hr/>
+{% if confirmed %}
+    <p>Your new email address is confirmed.</p>
```

Comment:
Can we include the email address in here?  E.g. "This confirms that the email address for your Zulip account has changed from x to y."